stage1/init/common/mount.go

```diff
@@ -342,6 +350,7 @@ func AppAddMounts(p *stage1commontypes.Pod, ra *schema.RuntimeApp, enterCmd []st
 			log.FatalE("Unable to setup app mounts", err)
```

Comment:
TODO in a followup PR right after; make everything here return err instead of FatalE-ing